frontend/src/services/http.ts

```diff
@@ -36,6 +41,11 @@ const doRequest = async <T>(url: string, method: string, body?: object | string)
   return Promise.resolve(result)
 }
 
+enum HttpStatusCode {
```

Comment:
And if some other function needs to use this enum? Now it's just inside of function scope. Can you put it outside this function, in this same file?